website/project/forms.py

```diff
@@ -16,4 +17,5 @@ class NewNodeForm(Form):
         validators.Length(max=200, message=u'Title must contain fewer than 200 characters.')
     ], widget=BootstrapTextInput())
     description = TextAreaField('Description', widget=BootstrapTextArea())
-    category = TextAreaField('Category', widget=BootstrapTextArea())
+    category = TextField('Category', widget=BootstrapTextInput())
+    inheritContributors = TextField('Inherit', widget=BootstrapTextInput())
```

Comment:
[Looks like](https://github.com/CenterForOpenScience/osf.io/blob/fcbc78f18f390218fd18af962c92e4d672d329b4/framework/forms/__init__.py#L3-L8) we're using [an [old version](https://github.com/CenterForOpenScience/osf.io/blob/fcbc78f18f390218fd18af962c92e4d672d329b4/requirements.txt#L13) of] [WTForms](http://wtforms.readthedocs.org/en/1.0.4/widgets.html).